x-pack/legacy/plugins/infra/public/pages/logs/analysis/page_setup_content.tsx

```diff
@@ -113,11 +113,17 @@ export const AnalysisSetupContent: React.FunctionComponent<AnalysisSetupContentP
                   <EuiText>
                     <FormattedMessage
                       id="xpack.infra.analysisSetup.errorText"
-                      defaultMessage="Please ensure your configured logs indices ({indexPattern}) exist. If your indices do exist, please try again."
+                      defaultMessage="Please ensure ALL configured logs indices ({indexPattern}) exist. If all these indices do exist, please try again."
```

Comment:
Are we sure the all-caps formatting is in line with the writing guide?

Reply:
I spotted this to, and I too would question the use of "ALL" in caps. It's better too avoid all caps unless there are exceptional circumstances. The straight error/info message is usually enough. 

I'd also suggest you could drop a few words in the rest of the message "If all these indices do exist" could become "If all indices exist".

@gchaps?

Reply:
I'd suggest:

Ensure all configured logs indices ({indexPattern}) exist, and try again.

In most cases, "please" is unnecessary. Exceptions are situations where the user must wait or do something inconvenient. Or, if the text sounds too abrupt without it.